test/input/yaml/Croton_NY_rfc.yaml

```diff
@@ -0,0 +1,129 @@
+                          # WARNING: `break_network_at_waterbodies: true` will only work if compute_method is set to "V02-structured-obj" and parallel_compute_method is unset (serial execution) or set to "by-network".
+    compute_method: V02-structured-obj  # OPTIONS: "V02-caching", "V02-structured-obj", "V02-structured"
+    assume_short_ts: true  # use the previous timestep value for both current and previous flow.
+    qts_subdivisions: 12  # number of timesteps per forcing (qlateral) timestep.
+    dt: 300  # default timestep length, seconds
+    nts: 288  # number of timesteps to simulate. If used with ql_file or ql_folder, nts must be less than the number of ql inputs x qts_subdivisions.
+    return_courant: false  # WARNING: true will only work with compute_method "V02-caching", therefore not currently compatible with simulation for waterbodies.
+#output file parameters
+output_parameters:
+    #output location for csv file
+    csv_output:
+        csv_output_folder: "../../test/output/text"
+        csv_output_segments: []
+    #out location for nc file
+    nc_output_folder: "../../test/output/text"
+#data column assignment inside supernetwork_parameters
+supernetwork_parameters:
+    title_string: "Croton1"
+    geo_file_path: "../../test/input/geo/NWM_2.1_Sample_Datasets/Croton_NY_TEST1/primary_domain/DOMAIN/Route_Link.nc"
+    #mask_file_path: ""
+    #mask_layer_string: ""
+    #mask_driver_string: "csv"
+    #mask_key: 0
+    columns:
+        key: "link"
+        downstream: "to"
+        dx: "Length"
+        n: "n"  # TODO: rename to `manningn`
+        ncc: "nCC"  # TODO: rename to `mannningncc`
+        s0: "So"  # TODO: rename to `bedslope`
+        bw: "BtmWdth"  # TODO: rename to `bottomwidth`
+        waterbody: "NHDWaterbodyComID"
+        tw: "TopWdth"  # TODO: rename to `topwidth`
+        twcc: "TopWdthCC"  # TODO: rename to `topwidthcc`
+        alt: "alt"
+        musk: "MusK"
+        musx: "MusX"
+        cs: "ChSlp"  # TODO: rename to `sideslope`
+    waterbody_null_code: -9999
+    terminal_code: 0
+    driver_string: NetCDF
+    layer_string: 0
+
+#waterbody parameters and assignments from lake parm file
+waterbody_parameters:
+    level_pool:
+        #WRF-Hydro lake parm file
+        level_pool_waterbody_parameter_file_path: "../../test/input/geo/NWM_2.1_Sample_Datasets/Croton_NY_TEST1/primary_domain/DOMAIN/LAKEPARM.nc"
+        level_pool_waterbody_id: lake_id
+        level_pool_waterbody_area: LkArea
+        level_pool_weir_elevation: WeirE
+        level_pool_waterbody_max_elevation: LkMxE
+        level_pool_outfall_weir_coefficient: WeirC
+        level_pool_outfall_weir_length: WeirL
+        level_pool_overall_dam_length: DamL
+        level_pool_orifice_elevation: OrificeE
+        level_pool_orifice_coefficient: OrificeC
+        level_pool_orifice_area: OrificeA
+
+    hybrid_and_rfc:
```

Comment:
@jdmattern-noaa @hellkite500 @mehdire61 
for the "hybrid" and "USGS" reservoirs, we need to build a list of the input files needed and summarize that in the documentation.